documentation/content/en/books/handbook/bsdinstall/_index.adoc

```diff
@@ -984,7 +984,7 @@ Here is a summary of the options that can be enabled in this menu:
 === Add Users
 
 The next menu prompts to create at least one user account.
-It is recommended to login to the system using a user account rather than as `root`.
+It is recommended to log in to the system using a user account rather than as `root`.
```

Comment:
I'm on the fence between "log in to" and "log into".

Reply:
Yeah, you're right. https://www.grammarly.com/blog/into-vs-in-to/